src/Stateless/MethodDescription.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+
+namespace Stateless
+{
+    /// <summary>
+    /// Describes a method - either an action (activate, deactivate, etc.) or a transition guard
+    /// </summary>
+    public class MethodDescription
+    {
+        readonly string _description;                     // _description can be null if user didn't specify a description
+
+        /// <summary>
+        /// Is the method synchronous or asynchronous?
+        /// </summary>
+        public enum Timing
```

Comment:
Since the class is now public, this should be `internal` or else it will form part of the public API.